feat: update icons to symbol

ndom91 opened the pull request:
## ☕️ Reasoning

- Coat of fresh paint :grin: 

## 🧢 Changes

- Update fileIcons to [Symbol](https://github.com/wilfriedago/vscode-symbols-icon-theme) icons. MIT Licensed VSCode icon pack
	- They have a public [figma](https://www.figma.com/file/HYLMyRbIdSbIJQlqnd9pSN/Symbols---File-Icons?node-id=20521%3A84115&t=PyBzZOlVG5TXyEdx-1)

Preview:

![image](https://github.com/gitbutlerapp/gitbutler/assets/7415984/8a1fcf26-a143-4304-92e8-a9ace9a4c454)

### Questions

@PavelLaptev I've kept the pre-existing `vsiIcons` files for now. What do you think, should we keep them in here or delete them?

<!--
If this PR is related to a specific issue, uncomment this section
and link it via the following text:

## 🎫 Affected issues

Fixes: INSERT_ISSUE_NUMBER

-->

<!--
If this is a WIP PR and you have todos left, feel free to uncomment this and turn this PR into a draft, see https://github.blog/2019-02-14-introducing-draft-pull-requests/

## 📌 Todos

-->

PavelLaptev reviewed (approved):
Looks good @ndom91, thank you! 
I've changed the icon size to 14px instead of 16 because the font size of the file name is also 14px, and the larger icons looked a bit too big.

PavelLaptev commented:
@ndom91 let's also remove old icons from our package

ndom91 commented:
> @ndom91 let's also remove old icons from our package

Removed the old ones - they still live in their own [repo](https://github.com/gitbutlerapp/file-extension-icon-JS) anyway, in case of any emergency haha 

Cleaned up the other few things as well. Should be good to go!
